CMakeLists.txt

```
@@ -1062,10 +1062,14 @@ else()
 	message(FATAL_ERROR "Either ENABLE_STATIC or ENABLE_SHARED has to be ON!")
 endif()
 
-macro(srt_add_program name)
+macro(srt_add_executable name)
```

A more descriptive name might be better here.
```suggestion
macro(srt_add_program_dont_install name)
```